lib/src/Pos/Node/API.hs

```diff
@@ -547,47 +553,209 @@ instance ToSchema (V1 Version) where
         pure $ NamedSchema (Just "V1Version") $ mempty
             & type_ .~ SwaggerString
 
+
+newtype SecurityParameter = SecurityParameter Int
+    deriving (Show, Eq, Generic, ToJSON, FromJSON, Arbitrary)
+
+instance Buildable SecurityParameter where
+    build (SecurityParameter i) = bprint shown i
+
+instance ToSchema SecurityParameter where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "SecurityParameter") $ mempty
+            & type_ .~ SwaggerNumber
+            & maximum_ .~ Just (fromIntegral (maxBound :: Int))
+            & minimum_ .~ Just (fromIntegral (minBound :: Int))
+
+
+instance ToSchema (V1 Core.SlotId) where
+    declareNamedSchema _ =
+        pure $ NamedSchema (Just "Core.SlotId") $ mempty
+            & type_ .~ SwaggerObject
+            & required .~ ["slot", "epoch"]
+            & properties .~ (mempty
+                & at "slot" ?~ (Inline $ mempty
+                    & type_ .~ SwaggerNumber
+                    & maximum_ .~ Just (fromIntegral (maxBound :: Word16))
+                    & minimum_ .~ Just (fromIntegral (minBound :: Word16))
+                    )
+                & at "epoch" ?~ (Inline $ mempty
+                    & type_ .~ SwaggerNumber
+                    & maximum_ .~ Just (fromIntegral (maxBound :: Word64))
+                    & minimum_ .~ Just (fromIntegral (minBound :: Word64))
+                    )
+                )
```

Comment:
same as above, you could just re-use schemas for `Word16` and `Word64`

```hs
declaredNamedSchema _ = 
    word64Schema <- declareNamedSchema (Proxy @Word64)
    word16Schema <- declareNamedSchema (Proxy @Word16)
    return $ NamedSchema ...
        & properties .~ (mempty
            & at "slot" ?~ word16Schema
            & at "epoch" ?~ word64Schema
```

Or alternatively, rely on the generic instances by using `genericSchemaDroppingPrefix`

```hs
  declareNamedSchema =
    genericSchemaDroppingPrefix "si" (\(--^) props -> props
      & ("slot"  --^ "<description of the slot>")
      & ("epoch" --^ "<description of the epoch>")
```